loqusdb/commands/export.py

```diff
@@ -51,7 +51,13 @@ def export(ctx, outfile, variant_type, freq):
     for chrom in existing_chromosomes:
         ordered_chromosomes.append(chrom)
 
-    nr_cases = adapter.case_count()
+    if variant_type == "snv":
+        nr_cases = adapter.nr_cases(snv_cases=True)
```

Comment:
On first glance, I might have put a couple of 
`{"$or": {"$exists": false}}` in the called function, here
https://github.com/Clinical-Genomics/loqusdb/blob/891ed9c7761512acc9f4312e938a9fe7cc73d2ba/loqusdb/plugins/mongo/case.py#L48. But it is actually pointless since there is an actual object model with a constructor. 💯